lib/rules/one-var.js

```diff
@@ -62,20 +120,46 @@ module.exports = function(context) {
         "FunctionDeclaration": startFunction,
         "FunctionExpression": startFunction,
         "ArrowFunctionExpression": startFunction,
+        "BlockStatement": startBlock,
```

Comment:
I'm confused. Isn't the `switch` test behaving exactly as it's supposed to? It should flag `let baz` with a message to combine the declaration with `let bar`, shouldn't it?